registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/client/AspspOutboundClient.java

```diff
@@ -0,0 +1,32 @@
+package de.adorsys.registry.manager.client;
+
+import de.adorsys.registry.manager.config.ClientConfiguration;
+import feign.*;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+
+@FeignClient(value = "aspsp-outbound-service", url = "${xs2a-adapter.url}",
+    configuration = ClientConfiguration.class)
+public interface AspspOutboundClient {
```

Comment:
in my opinion, it would be better to replace `outbound` by something more specified, because, in reality, we interact with xs2a-adapter

Reply:
if it's safe to expose that we communicate with Adapter (and it will request to Adapter only at any time in future), then ok.
Should I rebuild URIs as well, e.g. ```/v1/aspsps/adapter/import```?

Reply:
yes, please

Reply:
done